simulators/common/testmanager.go

```diff
@@ -189,11 +190,16 @@ func (manager *TestManager) ContainerIP(testSuite TestSuiteID, networkID, contai
 		return "", ErrNoSuchTestSuite
 	}
 	// if the containerID is "simulation", use simulation container ID
-	containerID, err := manager.isSimulation(containerID)
+	var err error
+	containerID, err = manager.isSimulation(containerID)
+	if err != nil {
+		return "", err
+	}
+	// if the networkID is "bridge", use bridge networkID
+	networkID, err = manager.isBridge(networkID)
```

Comment:
`isBridge` is not the best name, would be better to call it `findNetwork`.

Reply:
Also, I'm not sure if it will work. The idea with this function seems to be resolving the network ID from the network name, but `isBridge` just returns "bridge" for the input "bridge". Might be better to restructure things so the network ID is never required and all operations are by-name.